meson.build

```diff
@@ -14,6 +14,11 @@ project(
 add_project_arguments('-Wno-unused-parameter', language: 'c')
 add_project_arguments('-Wno-missing-braces', language: 'c')
 
+datadir = get_option('datadir')
+prefix = get_option('prefix')
+config = configuration_data()
```

Comment:
`config` isn't used anywhere, and can be removed

Reply:
addressed in bf442f0159c87de04e18429b3e5da123680a7069